src/js/constants.js

```diff
@@ -11,6 +11,8 @@ export const STUY_SPEC_API_HEADERS = {
 export const MEDIA_CREATOR_SLUGS = {
   illustration: "illustrators",
   photo: "photographers",
+  illustrators: "illustration",
```

Comment:
Shouldn't this be a different constant object?

Reply:
Nah, this can just act as a hardcoded doubly lookup.

Reply:
EHhhh. I guess

Reply:
Not a great name then

Reply:
Change to PROFILE_SLUGS

Reply:
@jkao1